src/sass/file-upload-input/file-upload-input.scss

```diff
@@ -0,0 +1,96 @@
+@import "../variables/variables";
+
+div.file-upload-input {
+    display: grid;
+    grid-template-rows: 1fr;
+    grid-template-columns: 1fr;
+    grid-template-areas: "center";
+}
+
+div.file-upload-input label[for="file-upload-input__input"] {
+    grid-area: center;
+}
+
+input.file-upload-input__input[type="file"] {
+    grid-area: center;
+    border-radius: var(--border-radius-100, 16px);
+}
+
+div.file-upload-input__container {
+    display: flex;
+    flex-direction: row;
+    background: var(--color-background-secondary, #f7f7f7);
```

Comment:
We should do
```
@include background-token(file-upload-input-background, color-background-secondary);
```
This should be done for some of those other colors as well
`file-upload-input-background` will not be defined, this is used for theming which we are planning on doing down the line.